druntime/src/core/memory.d

```diff
@@ -278,7 +278,11 @@ extern(C):
      * process footprint.  Collections may continue to occur in instances
      * where the implementation deems necessary for correct program behavior,
      * such as during an out of memory condition.  This function is reentrant,
-     * but enable must be called once for each call to disable.
+     * but `GC.enable()` must be called once for each call to `GC.disable()`.
+     * `GC.disable()` is preferred over the
```

Comment:
I would not say it is preferred, it just does a different thing. Maybe just explicitly say `GC.disable()` is how you reliably avoid GC pauses?

Reply:
ok I changed the wording